addons/udes_priorities/models/stock_picking.py

```diff
@@ -45,6 +48,14 @@ def _priority_group_domain(self, picking_type_ids=None):
         # To allow for this they are proxies to the default function
         return self._priority_and_priority_group_domain(picking_type_ids=picking_type_ids)
 
+    @api.depends("u_priority_id.sequence")
+    def _compute_priority_sequence(self):
+        """Set the sequence associated with a picking's priority."""
+        for record in self:
```

Comment:
Nit; maybe can filter out self with state done or cancel, and no need to check inside the loop after

Reply:
Good point!  Made the change